src/cli/serve.rs

```diff
@@ -88,7 +88,14 @@ fn show_start_message(bind_address: IpAddr, port: u16, color: ColorChoice) -> io
     write!(&mut buffer, "Visit ")?;
 
     buffer.set_color(&green)?;
-    write!(&mut buffer, "http://localhost:{}/", port)?;
+    if bind_address.is_loopback() {
+        write!(&mut buffer, "http://localhost:")?;
```

Comment:
The same loopback check is done to display the address earlier. You would be better off storing the address in a variable that is used for both displays IMO.